### ISSUE TITLE
Future of Alpaca/Roadmap

### ISSUE THREAD

**Issue by ryanmark** [gitana/alpaca#358]
Hey folks - yall have made an incredible tool here. It's become a critical component in a [project of ours](https://github.com/voxmedia/autotune). I can't thank you enough.

Up till now we've used Alpaca mostly as-is and sent y'all occasional bug fixes. It's looking likely that we will be doing more of that in the future, particularly around performance.

So I'm very curious what your long term plan is for Alpaca. If there are plans or goals for a 2.0 release? Is there a roadmap? Are there any big feature additions, architectural changes or bug fixes planned?

Basically, we want to send you more PRs, but want to make sure we're sending stuff that's in line with your plans.

**Comment by mteasdale1** [comment]
Hi Ryan

Thank you for your great support of Alpaca.  I will make some comments here and also followup directly to you on email:
- We (Cloud CMS) are committed to log term support for AlpacaJS as it is a critical component of our commercial product (Cloud CMS)
- We are also committed to Open Source for AlpacaJS
- About to release AlpacaJS 1.5.16 (probably within a week).  Our aim is to have an incremental release every 2 months.

**Comment by juniwalk** [comment]
@mteasdale1 Hello, I'd like to ask if you're planning to add form builder for use outside Cloud CMS. I just found AlpacaJS and the rendering portion is looking really great but I also need builder for my project.
